Examples/Scripts/Python/na60plus_detector.py

```diff
@@ -0,0 +1,130 @@
+# import the necessary modules
+
+import acts, acts.examples
+from acts.examples import geant4 as acts_g4
+
+
+from acts import (
+    svg,
+    logging,
+    Binning,
+    Extent,
+    GeometryContext,
+    ProtoBinning,
+    LayerStructureBuilder,
+    VolumeBoundsType,
+    VolumeStructureBuilder,
+    DetectorVolumeBuilder,
+    GeometryIdGenerator,
+    CuboidalContainerBuilder,
+    DetectorBuilder,
+    Transform3,
+    Range1D,
+    KdtSurfaces1D,
+    KdtSurfacesProvider1D,
+)
+
+geoContext = acts.GeometryContext()
+
+sensitive_matches = ["PixSensor"]
+passive_matches = ["PixStnFrame"]
+
+[elements, ssurfaces, psurfaces] = acts_g4.convertSurfaces(
+    "na60VT.gdml", sensitive_matches, passive_matches
+)
+
+logLevel = logging.VERBOSE
+
+# Write them to an obj file
+drawContext = acts.GeometryContext()
+sensitiveRgb = [0, 150, 150]
+passiveRgb = [150, 150, 0]
+
+kdtSurfaces = KdtSurfaces1D(geoContext, ssurfaces, [Binning.z])
+
+layers = [
+    [-386, -376],
+    [-376, -256],
+    [-256, -246],
+    [-246, -205],
+    [-205, -195],
+    [-195, -155],
+    [-155, -145],
+    [-145, -75],
+    [-75.0, -65],
+    [-65, 25],
+]
+
+volBuilders = []
+gaps = 0
+layerId = 0
+
+for il, lrange in enumerate(layers):
+    surfaces = kdtSurfaces.surfaces(Range1D(lrange))
```

Comment:
Can you remind me why we're using a KD tree here again, instead of just filtering the surfaces themselves? Do really need the speedup for geometry construction?